core/src/main/java/org/apache/iceberg/rest/RestException.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.rest;
+
+import com.google.errorprone.annotations.FormatMethod;
+
+public class RestException extends RuntimeException {
+  public RestException(String message) {
+    super(message);
+  }
+
+  @FormatMethod
+  public RestException(String message, Object... args) {
+    super(String.format(message, args));
+  }
+
+  public RestException(String message, Throwable cause) {
+    super(message, cause);
+  }
+
+  public RestException(Throwable cause) {
```

Comment:
We usually don't create constructors without a message because it allows people to be lazy later and not add context when handling exceptions.

Reply:
That's a good point. I'll remove this.